### DISCUSSION TITLE
Postgres Single Quote Escaping Breaking Migrations!

### DISCUSSION THREAD

**Discussion by ajhollowayvrm** [prisma/prisma#12080] [Q&A] [answered]
Hello, I am a very new user of Prisma.

My environment is as follows:
- Node JS and Prisms
- Posgresql

I am experimenting with Prisma Migrate and running into an error with JSONB and single quotes. 

Steps:
1. npx prisma db pull
`  birthday_award                        Json?    @default("{\"amount\": 250, \"status\": false, \"message\": \"HAPPY BIRTHDAY, @receiver_username!! Here''s +amount to celebrate!\", \"visibility\": \"Publicly Visible\"}")`

When I pull from the DB you can see the double single quotes ('') that Postgres uses to escape a single quote. This is just fine. 

2. Switch the connection to my staging DB and run `npx prisma migrate dev --name init` to try and copy the schema to my staging DB. I get this error pointing to the line in my schema.prisma above: 

    `DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState("42601"), message: "syntax error at or near \"\\\"", detail: None, hint: None, position: Some(Original(5069)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("scan.l"), line: Some(1180), routine: Some("scanner_yyerror") }`

I might be doing this whole thing wrong and thinking about it the wrong way too but can anyone shed some light into how to accommodate the double single quotes?

Thanks ahead of time for any responses!

**Comment by janpio** [comment, marked as answer]
No, that sounds like a real bug on our side. 

Did the CLI prompt you to submit an error report? If so, can you share the ID it gave you (or do this again to report the error, then share the ID)? Can you please also turn this into a bug report issue and supply all the information the template asks for? Then we can try to reproduce this as well and properly put it into our engineering process. Thanks!

**Reply by ajhollowayvrm** [reply]
https://github.com/prisma/prisma/issues/12095

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
